app/services/credit_notes/refunds/adyen_service.rb

```diff
@@ -30,7 +30,7 @@ def create
         refund.save!
 
         update_credit_note_status(refund.status)
-        track_refund_status_changed(refund.status)
+        Utils::Track.refund_status_changed(refund.status, credit_note.id, organization.id)
```

Comment:
```suggestion
        Utils::SegmentTrack.refund_status_changed(refund.status, credit_note.id, organization.id)
```

Reply:
what about using named parameters as well?

Reply:
I renamed the class.

I'm not sure about named arguments here, I find that it adds a lot fo noise:
```rb
something(refund.status, credit_note.id, organization.id)
# VS
something(status: refund.status, credit_note_id: credit_note.id, organization_id: organization.id
```